clients/app/src/views/components/base/Catch/Catch.tsx

```diff
@@ -13,17 +15,37 @@ const Catch: React.FC<ICatchProps> = ({
   count = 0,
   iconSize = 'ic-md',
   onPress = undefined,
+  textColor = 'buttonAction',
+  catchType = undefined,
+  catchResourceId = undefined,
 }) => {
+  const { pushTo } = useNavigationWithParams();
+
+  const onCountPress = React.useCallback(() => {
+    if (!catchType) {
+      return;
+    }
+
+    pushTo(commonScreens.profileList.name, {
+      id: catchResourceId,
+      type: catchType,
+    });
+  }, [catchType, catchResourceId, pushTo]);
+
   return (
-    <Pressable onPress={preventMultiCalls(onPress)} style={styles.container}>
+    <Pressable
+      onPress={preventMultiCalls(onPress)}
+      onLongPress={onCountPress}
```

Comment:
why do not use onPress only on number?

Reply:
on `Single Match` screen we have Catch button _(top right corner of screen)_ which are presented without number of catches.

I can change logic to:
- use long-press when the count of catches is hidden.
- use text press when the count of catches is visible.